src/server.c

```diff
@@ -5910,168 +5911,38 @@ int checkIgnoreWarning(const char *warning) {
 }
 
 #ifdef __linux__
-int linuxOvercommitMemoryValue(void) {
```

Comment:
p.s. aren't we also missing checkTcpBacklogSettings?

Reply:
This checks that the system can be configured according to the `tcp-backlog` configuration. So it's not a generic system check. What do you think?